pkg/event_handler/action_triggered_handler.go

```diff
@@ -102,7 +74,6 @@ func (eh ActionTriggeredHandler) getActionFinishedEvent(result keptnv2.ResultTyp
 			Result:  result,
 			Message: message,
 		},
-		Action: keptnv2.ActionData{},
```

Comment:
Why do we no longer need Action here?

Reply:
The `Action` field is not part of the `ActionFinishedEventData` anymore, so I had to remove it.